skipruntime-ts/adapters/postgres/package.json

```
@@ -0,0 +1,29 @@
+{
+  "name": "@skip-adapter/postgres",
+  "version": "0.0.2",
+  "type": "module",
+  "exports": {
+    ".": "./dist/index.js"
+  },
+  "scripts": {
+    "build": "tsc",
+    "clean": "rm -rf dist",
+    "lint": "eslint"
+  },
+  "author": "SkipLabs",
+  "license": "MIT",
+  "description": "Adapter providing reactive access to PostgreSQL from the Skip runtime",
+  "engines": {
+    "node": ">=22.6.0 <23.0.0"
+  },
+  "dependencies": {
+    "pg": "^8.13.1",
+    "pg-format": "^1.0.4",
+    "@skipruntime/core": "^0.0.5",
+    "@skipruntime/api": "^0.0.6"
```

These need to be updated after https://github.com/SkipLabs/skip/pull/668 right, and in any case want fixed versions:
```suggestion
    "@skipruntime/core": "0.0.5",
    "@skipruntime/api": "0.0.6"
```